### PR TITLE
Add bottom border to L screen navigation

### LINKED ISSUE #1150: Navigation light has lost a bottom border
## Issue Template

Navigation light has lost a bottom border

Screenshot:

screen shot 2017-07-05 at 12 08 13

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Add bottom border to L screen navigation

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/navigation/light/
- Verify bottom border on L breakpoints is correct

## Demo

http://vanilla-framework-1150-nav-btn-border.demo.haus/vanilla-framework/examples/patterns/navigation/light/

## Details

Fixes #1150

### CODE REVIEW

**Comment by barrymcgee** [comment]
@yaili This change also applies to dark navigation variant - it's unclear from the [spec](https://github.com/ubuntudesign/vanilla-design/tree/master/Navigation) if that should be the case or not, can you confirm?

File: scss/_patterns_navigation.scss [hunk 1]
```diff
@@ -40,6 +40,10 @@
   position: relative;
   width: 100%;
 
+  @media (min-width: $breakpoint-medium) {
+    border-bottom: 1px solid $border-color;
```
Comment (anthonydillon) [inline scss/_patterns_navigation.scss]: The spec says that only light navigation should have a border on small screens? Does this need updating? https://github.com/ubuntudesign/vanilla-design/blob/master/Navigation/navigation.md

File: scss/_patterns_navigation.scss [hunk 2]
```diff
@@ -25,6 +25,7 @@
       $border-color: $color-mid-light,
       $text-color: $color-dark
     );
+    border-bottom: 1px solid $color-mid-light;
```
Comment (barrymcgee) [inline scss/_patterns_navigation.scss]: Yes, all breakpoints - see: https://github.com/ubuntudesign/vanilla-design/blob/master/Navigation/navigation.png

As this is the exception, not the rule, I don't think an amendment to the mixin is necessary.

**Comment by yaili** [comment]
Can we remove the double bottom border when the small screen menu is open?

**Pull request merged**